gqrx.desktop

```diff
@@ -8,6 +8,7 @@ GenericName[ru]=Программно-определённое радио
 Comment[ru]=Приемник для программно-определенного радио (SDR) использующий GNU Radio и библиотеку Qt.
 GenericName[nl]=Software Defined Radio
 Comment[nl]=Software defined radio ontvanger geïmplementeerd met GNU Radio en de Qt GUI toolkit
+Commenr[de]=Software defined Radio implementiert durch GNU Radio und das Qt GUI Toolkit
```

Comment:
I'd suggest:
Comment[de]=Software Defined Radio auf Basis von GNU Radio und dem Qt GUI Toolkit

Sounds more natural to me.